src/js/va-letters/containers/ConfirmAddress.jsx

```diff
@@ -1,19 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import { updateAddressConfirmation } from '../actions';
+
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
 class ConfirmAddress extends React.Component {
+  constructor(props) {
+    super(props);
+    this.confirmAddress = this.confirmAddress.bind(this);
+  }
+
+  confirmAddress() {
+    this.props.onConfirmationOfAddress(true);
+    this.props.router.push('/download-letters');
+  }
+
   render() {
+    const buttons = this.props.letters.addressConfirmed !== false ?
+        (<div className="usa-grid-full form-progress-buttons">
+          <div className="usa-width-one-fourth">
+            <ProgressButton
+                onButtonClick={this.confirmAddress}
+                buttonText="Yes"
+                buttonClass="usa-button-primary"/>
+          </div>
+          <div className="usa-width-one-fourth">
+            <ProgressButton
+                onButtonClick={() => this.props.onConfirmationOfAddress(false)}
```

Comment:
I don't think I fully understand. The reason I had to do `() => ...` was because without that `this.props.onConfirmationOfAddress(false)` would be executed immediately instead of passing the function as a prop to `ProgressButton` to be executed `onClick`. I don't think I can bind a function that's part of `this.props`, since that is read only.